ansible/vars/topo_dual_t0.yml

```diff
@@ -0,0 +1,204 @@
+topology:
+  host_interfaces:
+    - 0.0,1.0
```

Comment:
ptf is connecting to both tor, here we need to emulate the Y-cable using openvswitch.